homeassistant/components/justnimbus/sensor.py

```diff
@@ -104,57 +69,46 @@ class JustNimbusEntityDescription(
         icon="mdi:car-coolant-level",
         native_unit_of_measurement=UnitOfVolume.LITERS,
         device_class=SensorDeviceClass.VOLUME,
-        state_class=SensorStateClass.MEASUREMENT,
+        state_class=SensorStateClass.TOTAL,
         entity_category=EntityCategory.DIAGNOSTIC,
         value_fn=lambda coordinator: coordinator.data.reservoir_content,
     ),
     JustNimbusEntityDescription(
-        key="total_saved",
-        translation_key="total_saved",
+        key="water_saved",
+        translation_key="water_saved",
         icon="mdi:water-opacity",
         native_unit_of_measurement=UnitOfVolume.LITERS,
         device_class=SensorDeviceClass.VOLUME,
         state_class=SensorStateClass.TOTAL_INCREASING,
         entity_category=EntityCategory.DIAGNOSTIC,
-        value_fn=lambda coordinator: coordinator.data.total_saved,
+        value_fn=lambda coordinator: coordinator.data.water_saved,
     ),
     JustNimbusEntityDescription(
-        key="total_replenished",
-        translation_key="total_replenished",
-        icon="mdi:water",
-        native_unit_of_measurement=UnitOfVolume.LITERS,
-        device_class=SensorDeviceClass.VOLUME,
-        state_class=SensorStateClass.TOTAL_INCREASING,
-        entity_category=EntityCategory.DIAGNOSTIC,
-        value_fn=lambda coordinator: coordinator.data.total_replenished,
-    ),
-    JustNimbusEntityDescription(
-        key="error_code",
-        translation_key="error_code",
-        icon="mdi:bug",
-        entity_registry_enabled_default=False,
-        entity_category=EntityCategory.DIAGNOSTIC,
-        value_fn=lambda coordinator: coordinator.data.error_code,
-    ),
-    JustNimbusEntityDescription(
-        key="totver",
-        translation_key="total_use",
+        key="water_used",
```

Comment:
We need to add a breaking change paragraph for removed and added sensors. Explain what the user needs to do to cope with the changes. The user can delete the old entities after updating any automations or scripts that use them.